library/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/processor/WireframeUtils.kt

```diff
@@ -72,6 +72,13 @@ internal class WireframeUtils {
         }
     }
 
+    private fun MobileSegment.Wireframe.hasBackground(): Boolean {
```

Comment:
what if the background is transparent?

Reply:
Good catch, I am going to filter those out also.

Reply:
👍 or semi-transparent. On iOS we recently added [`isTranslucent`](https://github.com/DataDog/dd-sdk-ios/blob/15b5accc7b9bfa65228b5c69205a055dd46c2ec7/session-replay/Sources/DatadogSessionReplay/Recorder/ViewTreeSnapshotProducer/ViewTreeSnapshot/ViewTreeSnapshot.swift#L96-L99) for similar use case, which also takes view's `alpha` (opacity) into account. It doesn't yet check the background color's alpha, but remains extensible in that direction (e.g. a background of `#FF000099` should be considered translucent).

Reply:
yup I modified it in order to just consider alpha `FF` in the color and `1f` in the `ShapeStyle` opacity. Any other wireframe with any alpha value will be not considered as overlay.